.github/workflows/black.yml

```diff
@@ -0,0 +1,71 @@
+name: black
+on:
+  workflow_dispatch:
+  schedule:
+    - cron: "0 0 * * *"
+
+jobs:
+  format:
+    runs-on: windows-latest
+    steps:
+      - name: Checkout the repository
+        uses: actions/checkout@v2
+        with: 
+          fetch-depth: 0
+  
+      - name: Set up Python
+        uses: actions/setup-python@v2
+        with:
+          python-version: 3.8
+
+      - name: Install black
+        run: pip install black
+
+      - name: Black
+        run: black manim tests example_scenes
+
+      - name: Check for modified files
+        shell: bash
+        id: git-check
+        run: echo ::set-output name=modified::$(if git diff-index --quiet HEAD --; then echo "false"; else echo "true"; fi)
+
+      - name: Print Black Version if Failed
+        shell: bash
+        if: steps.git-check.outputs.modified == 'true'
+        run: |
+          blackVer=$(black --version)
+          blackVersion=$(python -c "print('${blackVer}'.split()[2])")
+          date=$(python -c "import time;print(time.localtime().tm_mday)")
+          pushBranch="black-${blackVersion}-${date}"
+          git config user.name "github-actions"
+          git config user.email "actions@github.com"
+          git checkout -B "$pushBranch"
+          git add .
+          git commit -m "Running Black ON Master"
+
+      - name: Push branch
+        if: steps.git-check.outputs.modified == 'true'
+        shell: bash
+        run: |
+          blackVer=$(black --version)
+          blackVersion=$(python -c "print('${blackVer}'.split()[2])")
+          date=$(python -c "import time;print(time.localtime().tm_mday)")
+          pushBranch="black-${blackVersion}-${date}"
+          git push -u origin "${pushBranch}" --force
+
+      - name: Make a PR
+        env:
+          access_token: ${{ secrets.GITHUB_TOKEN }}
+        run: |
+          $env:GITHUB_TOKEN = $env:access_token
+          $blackVer = black --version
+          $blackVersion = python -c "print('$($blackVer)'.split()[2])"
+          $date = python -c "import time;print(time.localtime().tm_mday)"
+          $pushBranch="black-$($blackVersion)-$($date)"
+          $postContents = @{title="Black on Master - Automation";head="ManimCommunity:$($pushBranch)";base="master";body="This an automated pull requests."} | ConvertTo-Json -Compress
```

Comment:
Maybe suggest me some good `title` and `body` :).

Reply:
Maybe "updating Black to the latest version available [{version})" ? 

And for the body, I don't know, a more descriptive thing, like what is this PR, why etc.

Reply:
Wait I'm confused. This is for the PR, no? The PR doesn't update the black version, it only runs black and commits the differences, correct?

If so, the title could be "Automated PR: autoformat with black". The body may contain the list of files changed, if that's at all possible. If not, just leave the body empty.

Reply:
Please wait for me to do some changes here. I will make if more readable.